modules/environment-roles/templates/shared_iam_terraform_policy.json.tpl

```diff
@@ -31,10 +31,9 @@
         "arn:aws:iam::${account_id}:policy/TDRCreateDbUsersConsignmentapiPolicy${title(environment)}",
         "arn:aws:iam::${account_id}:policy/TDRCreateKeycloak*",
         "arn:aws:iam::${account_id}:policy/TDRCustodian*",
-        "arn:aws:iam::${account_id}:policy/TDRDbMigrationLambdaPolicy${title(environment)}",
+        "arn:aws:iam::${account_id}:policy/*Lambda*${title(environment)}",
```

Comment:
Could we keep the `TDR`, just to control the access a bit more, eg: `.../TDR*Lambda*...`

Reply:
Yeah we can, I checked the new roles and policies and they are called TDRxxx. I've changed it and the same for the roles below.